Common/src/main/resources/assets/biomeswevegone/lang/ru_ru.json

```
@@ -487,12 +508,21 @@
   "block.biomeswevegone.overgrown_stone": "Заросший камень",
   "block.biomeswevegone.packed_black_ice": "Упакованный чёрный лёд",
   "block.biomeswevegone.packed_borealis_ice": "Упакованный арктический лёд",
+  "block.biomeswevegone.packed_pale_mud": "Упакованная бледная грязь",
+  "block.biomeswevegone.pale_jack_o_lantern": "Бледный светильник Джека",
+  "block.biomeswevegone.pale_mud": "Бледная грязь",
+  "block.biomeswevegone.pale_mud_bricks": "Кирпичи из бледной грязи",
+  "block.biomeswevegone.pale_mud_bricks_slab": "Плита из кирпичей бледной грязи",
```

Всё, что сделано из кирпичей должно переводиться как "... из ... кирпича". Это касается, плит, ступенек и оград.
Для сложных слов допускается составная конструкция. Два слова "из" не допускаются.
```suggestion
  "block.biomeswevegone.pale_mud_bricks_slab": "Плита из бледно-грязевого кирпича",
```